config/socket/game.js

```diff
@@ -179,9 +192,11 @@ Game.prototype.selectFirst = function() {
     this.players[winnerIndex].points++;
     this.winnerAutopicked = true;
     this.stateResults(this);
+    this.sendNotification(this.players[winnerIndex].username+' has won the round!');
+    this.sendUpdate();
   } else {
     // console.log(this.gameID,'no cards were picked!');
```

Comment:
I'll remove the ones that are commented out. 

Some of are actually useful.

Done